content/_data/changelogs/weekly.yml

```diff
@@ -1009,6 +1009,29 @@
       pull: 2984
       issue: 46288
     # pull: 2981 not notable
+- version: "2.76"
+  date: 2017-08-27
+  changes:
+    - type: rfe
+      message: >
+        Update remoting from 3.10 to 3.11 to improve stability and diagnosability.
+      references:
+        - pull: 2988
+        - url: https://github.com/jenkinsci/remoting/blob/master/CHANGELOG.md#311
```

Comment:
I propose to stick to my changelog proposals in https://github.com/jenkinsci/jenkins/pull/2988, because...

* JENKINS-46259, JENKINS-45233, JENKINS-45023  are LTS Candidates. Not referencing them in this changelog may cause confusion
* JENKINS-37567 is important due to the notice of possible regressions in Java Web Start agents

I can leave without referencing other entries

Reply:
@oleg-nenashev Thanks, I created #1108 to follow the usual PR pattern and mention the issues (but without dedicated entries).